microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -163,13 +164,40 @@ Status:
 To capture JavaScript stack traces when DOM nodes are added to the DOM at runtime, enable this experiment. The captured stack traces are displayed in the **Stack Trace** pane of the **Elements** panel.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 113.
-*  This checkbox is present in Microsoft Edge Stable 111.
+*  This checkbox is present in Microsoft Edge Canary 115.
+*  This checkbox is present in Microsoft Edge Stable 112.
+
+
+<!-- ====================================================================== -->
```

Comment:
@captainbrosset @zoherghadyali - does this Prettyprint Minified experiment now affect tools other than Sources?  checkbox label changed.

starting w/ 115, checkbox label is now:
Automatically pretty print minified sources
instead of:
Automatically pretty print in the Sources Panel

The label now mentions _what's_ prettyprinted (minified sources)  instead of _where_ that's done (Sources panel).

Reply:
kept as-is